variables.tf

```diff
@@ -12,7 +12,7 @@ variable "subnet_group_name" {
 }
 
 variable "parameter_group_name" {
-  default = "default.memcached1.4"
+  default = "default.memcached1.5"
```

Comment:
This would be a breaking change (desired state changes) no?

Reply:
This is due to the default version for memcached changing. We could lock `engine_version` to 1.4 but that might change desired state too.

Reply:
Should we leave this in and document the changes?

Reply:
I think that sounds reasonable, if default version has changed then it will be a breaking change either way right?

Reply:
Yepp it will, I'm going to leave it in and make note in the release.